src/components/ProfileDropdown.tsx

```diff
@@ -14,6 +19,25 @@ const ProfileDropdown = ({ image }: ProfileDropdownProps) => {
 		setIsOpen(!isOpen);
 	};
 
+	const dispatch = useAppDispatch();
+	const { isLoading } = useAppSelector((state) => state.logout);
+	const { showErrorMessage, showWorningMessage } = useToast();
+
+	const handleLogout = () => {
+		try {
+			dispatch(Logout());
+			showWorningMessage("You've logged out!");
+		} catch (error) {
```

Comment:
let's display messages from backend not hardcoded one, same case on error, always unwrap your dispatch to  avoid unnecessary issues.

Reply:
Done